speechbrain/core.py

```diff
@@ -719,12 +720,13 @@ def _train_loader_specifics(self, dataset, loader_kwargs):
                 self.train_sampler = DistributedSamplerWrapper(
                     sampler,
                     rank=self.rank,
-                    drop_last=drop_last,
-                    shuffle=shuffle,
+                    # drop_last=drop_last,
```

Comment:
I understand why shuffle is dropped. But what about drop_last ?

Reply:
no clue. I think it was this way somewhere? not on purpose...